packages/schema/src/config/postcss.ts

```diff
@@ -11,29 +9,6 @@ export default defineUntypedSchema({
      * @type {Record<string, any>}
      */
     plugins: {
-      /**
-       * https://github.com/postcss/postcss-import
-       */
-      'postcss-import': {
-        $resolve: async (val, get) => val !== false
-          ? defu(val || {}, {
-            resolve: createResolver({
-              alias: { ...(await get('alias')) },
-              modules: [
-                await get('srcDir'),
-                await get('rootDir'),
-                ...(await get('modulesDir'))
-              ]
-            })
-          })
-          : val
-      },
-
-      /**
-       * https://github.com/postcss/postcss-url
-       */
-      'postcss-url': {},
-
```

Comment:
Shouldnt we leave the type as is and mark it as deprecated to avoid breaking changes?